js/test/Exchange/test.signIn.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+// ----------------------------------------------------------------------------
+
+async function testSignIn(exchange) {
+    const method = 'signIn';
+    if (exchange.has[method]) {
+        await exchange[method] ();
+        console.log (method + ' successful');
+    } else {
+        console.log (method + '() is not supported');
```

Comment:
this message is not very meaningful because it gives the idea that it's a missing feature 🤔

Reply:
Would be nice to mention how you think it to be, or edit directly. but as it will need some more minutes for replies back and forth, before you reply, I'll edit msg, and you can either fully remove `else` clause, or edit the message as you think it could be better👍

Reply:
@carlosmiei did you have a chance to re-view this?